vic/drivers/image/src/get_global_param.c

```diff
@@ -777,6 +777,9 @@ get_global_param(FILE *gp)
             else if (strcasecmp("SKIPYEAR", optstr) == 0) {
                 sscanf(cmdstr, "%*s %hu", &global_param.skipyear);
             }
+            else if (strcasecmp("COORD_DIMS_OUT", optstr) == 0) {
+                sscanf(cmdstr, "%*s %zu", &options.COORD_DIMS_OUT);
+            }
```

Comment:
Sorry I don't understand what you mean. Do you mean the usage of COORD_DIMS_OUT in the global_param file? I remember we talked about it when we were discussing the conventions. We then agreed implement it in the global_param file. Do you have any suggestions about how to get this info from the domain file (how many dimensions the output of the lat/lon values should have)?
Just to explain the statement COORD_DIMS_OUT:
COORD_DIMS_OUT = 1 gives 1D latlon values in the output files. ncdump example of an output file: 
        double lon(lon) ;
                lon:long_name = "longitude" ;
                lon:units = "degrees_east" ;
                lon:standard_name = "longitude" ;
        double lat(lon) ;
                lat:long_name = "latitude" ;
                lat:units = "degrees_north" ;
                lat:standard_name = "latitude" ;

COORD_DIMS_OUT = 2 gives 2D latlon values in the output files. ncdump example of an output file: 
        double lon(lat, lon) ;
                lon:long_name = "longitude" ;
                lon:units = "degrees_east" ;
                lon:standard_name = "longitude" ;
        double lat(lat, lon) ;
                lat:long_name = "latitude" ;
                lat:units = "degrees_north" ;
                lat:standard_name = "latitude" ;